module/dice/damage-roll.mjs

```diff
@@ -298,4 +298,14 @@ export default class DamageRoll extends BasicRoll {
       { critical: { allow: allowCritical } }, { options: { title } }, { rollMode: defaultRollMode }
     );
   }
+
+  /* -------------------------------------------- */
+  /*  Evaluate Methods                            */
+  /* -------------------------------------------- */
+
+  /** @inheritDoc */
+  async evaluate(options={}) {
+    this.options.appearance = { colorset: this.options.type };
```

Comment:
It would probably be good to have this behind a check and to use `setProperty` to avoid deleting any existing appearance options:

```suggestion
    if ( this.options.type && !this.options.appearance?.colorset ) {
      foundry.utils.setProperty(this.options, "appearance.colorset", this.options.type);
    }
```